ACE/ace/OS_NS_sys_socket.h

```diff
@@ -65,9 +65,7 @@
 #endif
 
 #if defined (ACE_HAS_IPV6)
-#if defined (IPV6_PKTINFO)
-#define ACE_RECVPKTINFO6 IPV6_PKTINFO
-#elif defined (IPV6_RECVPKTINFO)
+#if defined (IPV6_RECVPKTINFO)
```

Comment:
Why remove the lines above?

Reply:
They incorrectly define `ACE_RECVPKTINFO6` to `IPV6_PKTINFO` which is a tag used to identify the packet info as opposed to a socket option.